packages/context/src/objects/objects.ts

```diff
@@ -43,6 +43,37 @@ export type ObjectInstance = {
     [key: string]: unknown;
 };
 
+export type FormioInputTypes = {
+    label: string;
+    type: string;
+    key: string;
+    initialValue?: any;
+    property?: Property;
```

Comment:
Out of curiosity, what is this used for?  What sets this in Builder?

Reply:
In the builder, whenever we drag and drop a field in the form this `property` automatically gets added to the JSON via the code.

Reply:
I believe this could be retrieved from the `objectProperties` if we don't set it in the builder. This can be removed.